kqueen_ui/config/base.py

```diff
@@ -56,7 +57,8 @@ class BaseConfig:
     PROVISIONER_UNKNOWN_STATE = 'Not Reachable'
 
     # Authentication choices
-    AUTH_OPTIONS = {}
+    LDAP_AUTH_NOTIFY = False
```

Comment:
In the rest of the file boolean variables are strings, let's keep consistent and dot he same for those paramaters

Reply:
ok, so need to fix     ENABLE_PUBLIC_REGISTRATION = 'False' in that file in a separate change

Reply:
ok,  so pls approve changes @atengler @katyafervent